packages/ckeditor5-utils/src/env.ts

```diff
@@ -109,7 +111,9 @@ const env: EnvType = {
 
 	isBlink: isBlink( userAgent ),
 
-	isMediaForcedColors: isMediaForcedColors(),
+	get isMediaForcedColors() {
+		return isMediaForcedColors();
+	},
```

Comment:
I first wanted to get rid of these getters and make it so that the property is resolved immediately (just like all other `is*` properties above). However, the existing tests were failing and it appeared that we test whether the util reacts to the change of related settings.

Indeed, this setting can change (tested that), so I decided to keep these functions as getters.

Reply:
Doh... so maybe this should be a function? To indicate that it's a dynamic value?

Reply:
All the other properties of the `env` object are static (should not change with time) except the two recently introduced. So it'd make sense to think twice before we commit in any direction.